pkg/agent/route/route.go

```diff
@@ -36,10 +37,22 @@ const (
 	routeTableConfigPath  = "/etc/iproute2/rt_tables"
 	// AntreaIPRulePriority is Antrea IP rule priority
 	AntreaIPRulePriority = 300
+	defaultGWIP          = "169.254.253.1"
```

Comment:
Let us use another name to differentiate with default GW (gw0), like virtualGWIP/MAC.

Please add some comments to explain them, and mention they are for policy-only mode.

Reply:
done.